packages/lesswrong/lib/routes.ts

```diff
@@ -372,187 +376,150 @@ addRoute(
   }
 );
 
-if (taggingNameIsSet.get()) {
-  addRoute(
-    {
-      name: 'tagsSingleCustomName',
-      path: `/${taggingNamePluralSetting.get()}/:slug`,
-      componentName: 'TagPageRouter',
-      titleComponentName: 'TagPageTitle',
-      subtitleComponentName: 'TagPageTitle',
-      previewComponentName: 'TagHoverPreview',
-      enableResourcePrefetch: tagRouteWillDefinitelyReturn200,
+addRoute(
```

Comment:
FYI to EA forum: Along with the changes to `instanceSettings`, these changes are probably among the highest-risk things to test.